<?xml version="1.0" encoding="UTF-8"?>
<testsuites name="Mocha Tests" time="19.068" tests="1" failures="1">
  <testsuite name="Root Suite" timestamp="2022-10-11T07:30:10" tests="0" file="cypress/e2e/onliner.cy.ts" time="0.000" failures="0">
  </testsuite>
  <testsuite name="Test Suite" timestamp="2022-10-11T07:30:15" tests="1" time="13.849" failures="1">
    <testcase name="Test Suite First test" time="0.000" classname="First test">
      <failure message="Timed out retrying after 4000ms: expected &apos;&lt;button.auth-button.auth-button_primary.auth-button_middle.auth-form__button.auth-form__button_width_full&gt;&apos; to contain text &apos;Войти1&apos;, but the text was &apos;\n                Войти\n            &apos;" type="AssertionError"><![CDATA[AssertionError: Timed out retrying after 4000ms: expected '<button.auth-button.auth-button_primary.auth-button_middle.auth-form__button.auth-form__button_width_full>' to contain text 'Войти1', but the text was '\n                Войти\n            '
    at Context.eval (webpack:///./cypress/e2e/onliner.cy.ts:10:64)

      + expected - actual

      -'\n                Войти\n            '
      +'Войти1'
      ]]></failure>
    </testcase>
  </testsuite>
</testsuites>